config/environments/development.rb

```diff
@@ -15,6 +15,7 @@
 
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.perform_deliveries = false
```

Comment:
Cheers, this was just a stop gap to test changes without bothering anyone else in the company. :) Letter Opener is a much better alternative though.